net-misc/cadaver/cadaver-0.23.3.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="A command line WebDAV client"
+DESCRIPTION="It supports file upload, download, on-screen display, namespace ope
+rations (move/copy), collection creation and deletion, and locking operations."
+HOMEPAGE="http://www.webdav.org/cadaver"
+COPYRIGHT="1998-2008 Joe Orton,
+   except where otherwise indicated.
+   2002-2003 GRASE Lab, UCSC."
```

Comment:
I'd say we can drop the comment, a comma and maybe the full stop:
```diff
-COPYRIGHT="1998-2008 Joe Orton,
+COPYRIGHT="1998-2008 Joe Orton
-   except where otherwise indicated.
-   2002-2003 GRASE Lab, UCSC."
+	2002-2003 GRASE Lab, UCSC"
```
BTW, the second line (of [`COPYRIGHT`](https://github.com/haikuports/haikuports/wiki/HaikuPorter-BuildRecipes#copyright)) should have a tab.